src/Sentry/Platforms/Android/SentrySdk.cs

```diff
@@ -59,6 +59,11 @@ private static void InitSentryAndroidSdk(SentryOptions options)
 
         // Define the configuration for the Android SDK
         SentryAndroidOptions? nativeOptions = null;
+
+        // Convert NdkHandlerStrategy .net to Java Android
+        var handlerStrat = (int)options.HandlerStrategy;
+        JavaSdk.Android.Core.NdkHandlerStrategy? handlerStrategy = JavaSdk.Android.Core.NdkHandlerStrategy.Values()?[handlerStrat];
```

Comment:
I'm struggling to parse. Why would `Values()` return null?
Can we use `var`
```suggestion
        var handlerStrategy = JavaSdk.Android.Core.NdkHandlerStrategy.Values()[(int)options.HandlerStrategy];
```

An extension method on `NdkHandlerStrategy` is probably a good solution:

```csharp
ToJava()
```

Reply:
I'll try the `ToJava()` thanks for the suggestion

Reply:
> Why would Values() return null?

@bruno-garcia In my experience, it's always better to assume that a method can return null, even if it seems impossible. I quite feel strongly about it. 
We would rather get it to default to `SENTRY_HANDLER_STRATEGY_DEFAULT` than throw an unhandled exception which would cause things to break somewhere else that would be harder to troubleshoot in 10 months. `Values()?` doesn't have any cost compared to `Values()`. 

Now `JavaSdk.Android.Core.NdkHandlerStrategy?` is silly, let's do a JavaSdk.Android.Core.NdkHandlerStrategy.Values()?[handlerStrat] ?? JavaSdk.Android.Core.NdkHandlerStrategy.SENTRY_HANDLER_STRATEGY_DEFAULT

Reply:
> We would rather get it to default to SENTRY_HANDLER_STRATEGY_DEFAULT than throw an unhandled exception which would cause things to break somewhere else that would be harder to troubleshoot in 10 months. Values()? doesn't have any cost compared to Values().

It's more nuanced than that. With the `??` approach we also will scratch our heads when things are not working the way we expected: "I set the handler to A, why is B happening"?

You should add in code or comments that intention you describe in this comment then. Why would it be null in this case? Is some possible bindings problem? Is it the caller could be on a version of C# pre-nullability?
Once you clarify why that could be the case, you can use our debug logger. Try to read the value and if it's null you use `LogError("expected a value set for handler since a default is hard coded but got null")

IMO that's overkill, and throwing `??` all over the code base "just in case" is far from ideal.

That said I share the sentiment that we can't foresee all issues and this is a crash reporting SDK so we need ot be careful. The way we go about that is by using error handling and our debug logging. You could do that on `ToJava` only, and avoid the other `??` usage. `ToJava` would require the internal `ILogger` as an argument to properly log out the error. But unless there's a reasonable suspicion (based on experience or otherwise) that reading that value could be null even though we're hard coding values all over the place, it's enough to have a higher level try/catch deal (debug logging, always) with it and avoid the app from crashing.

Reply:
fair enough, you're right